lib/publiccloud/instance.pm

```diff
@@ -429,7 +429,7 @@ sub softreboot {
     }
 
     $self->ssh_assert_script_run(cmd => 'sudo /sbin/shutdown -r +1');
-    sleep 60;    # wait for the +1 in the previous command
+    sleep 180;    # wait for the +1 in the previous command, wait for the system to turn off
```

Comment:
Please make this conditional so if it's SAP we'll wait 3 minutes otherwise just 1 minute.

Reply:
Sure.